### PR TITLE
Upgrade evmone to 0.4.1, use and parse json output

### PR DESCRIPTION
**Pull request by hugo-dc**
Fixes https://github.com/ewasm/benchmarking/issues/72

### CODE REVIEW

File: evm/evmone/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN pip3 install durationpy jinja2 pandas
 
 # install evmone
 WORKDIR /root
-RUN git clone --recursive --single-branch --branch bench-evm-codes https://github.com/ewasm-benchmarking/evmone
+RUN git clone --recursive --single-branch --branch v0.4.1-benchmarking https://github.com/ewasm-benchmarking/evmone
```
Comment (axic) [inline evm/evmone/Dockerfile:12]: It seems v0.4.1-benchmarking is identical to v0.4.1. Shouldn't we just use upstream as-is in this case?

**Comment by axic** [comment]
Please update the README too.

**Comment by hugo-dc** [comment]
@axic updated README and used `v0.4.1` instead of new/identical `v0.4.1-benchmarking`.

File: README.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ $ make notebook
 ## EVM engine versions
 
 - cita-vm: v0.1.3 (May 2019)
-- evmone: 94f4e827 (closest to v0.1.0)
+- evmone: 66d8b1e6 (v0.4.1)
```
Comment (axic) [inline README.md:135]: Just use v0.4.1, no need for hash if it is an upstream release.

File: evm/evmone/Dockerfile [hunk 3]
```diff
@@ -9,7 +9,7 @@ RUN pip3 install durationpy jinja2 pandas
 
 # install evmone
 WORKDIR /root
-RUN git clone --recursive --single-branch --branch bench-evm-codes https://github.com/ewasm-benchmarking/evmone
+RUN git clone --recursive --single-branch --branch v0.4.1 https://github.com/ewasm-benchmarking/evmone
```
Comment (axic) [inline evm/evmone/Dockerfile:12]: If we are using upstream:
```suggestion
RUN git clone --recursive --single-branch --branch v0.4.1 https://github.com/ethereum/evmone
```

**Comment by hugo-dc** [comment]
fixed

File: evm/evmone/Dockerfile [hunk 4]
```diff
@@ -9,7 +9,7 @@ RUN pip3 install durationpy jinja2 pandas
 
 # install evmone
 WORKDIR /root
-RUN git clone --recursive --single-branch --branch bench-evm-codes https://github.com/ewasm-benchmarking/evmone
+RUN git clone --recursive --single-branch --branch v0.4.1 https://github.com/ethereum/evmone
 RUN cd evmone && mkdir build
 RUN cd evmone/build && CXXFLAGS="-Wno-error=deprecated-copy" cmake .. -DEVMONE_TESTING=ON
 RUN cd evmone/build && cmake --build . -- -j
```
Comment (axic) [inline evm/evmone/Dockerfile:15]:
```suggestion
RUN cd evmone/build && make -j4
```

File: evm/evmone/Dockerfile [hunk 5]
```diff
@@ -9,7 +9,7 @@ RUN pip3 install durationpy jinja2 pandas
 
 # install evmone
 WORKDIR /root
-RUN git clone --recursive --single-branch --branch bench-evm-codes https://github.com/ewasm-benchmarking/evmone
+RUN git clone --recursive --single-branch --branch v0.4.1 https://github.com/ethereum/evmone
 RUN cd evmone && mkdir build
 RUN cd evmone/build && CXXFLAGS="-Wno-error=deprecated-copy" cmake .. -DEVMONE_TESTING=ON
```
Comment (axic) [inline evm/evmone/Dockerfile:14]: Is `CXXFLAGS="-Wno-error=deprecated-copy"` still needed?

Comment (hugo-dc) [reply evm/evmone/Dockerfile]: It is no longer needed, I've removed it.

**Comment by axic** [comment]
Can someone rerun the evm benchmarks so we can see hoe it changed?

**Comment by hugo-dc** [comment]
Here is a comparison of current version of evmone in `master` vs evmone in this PR (0.4.1): https://gist.github.com/hugo-dc/96e15180e463ef455a90e0475a44b36b , for most of the benchmarks evmone `0.4.1` is almost 2x faster.

**Comment by axic** [comment]
> for most of the benchmarks evmone 0.4.1 is almost 2x faster.

Looks great! cc @chfast @cdetrio

**Pull request merged**